GitCommands/Git/GitModule.cs

```diff
@@ -2172,6 +2172,53 @@ public IReadOnlyList<GitStash> GetStashes(bool noLocks = false)
             return stashes;
         }
 
+        public async Task<ExecutionResult> GetSingleDifftoolAsync(
+            ObjectId? firstId,
+            ObjectId? secondId,
+            string? fileName,
+            string? oldFileName,
+            ArgumentString extraDiffArguments,
+            bool cacheResult,
+            bool isTracked,
+            bool useGitColoring,
+            CancellationToken cancellationToken)
+        {
+            // fix refs slashes
+            fileName = fileName.ToPosixPath();
+            oldFileName = oldFileName.ToPosixPath();
+            string? firstRevision = firstId?.ToString().ToPosixPath();
+            string? secondRevision = secondId?.ToString().ToPosixPath();
```

Comment:
```suggestion
            string? firstRevision = firstId?.ToString();
            string? secondRevision = secondId?.ToString();
```

Reply:
This was copied from GetSingleDiffAsync(), changed there too.